src/engines_gpl/dflowfm/packages/dflowfm_kernel/src/dflowfm_data/unstruc_model.f90

```diff
@@ -2869,7 +2869,7 @@ subroutine writeMDUFilepointer(mout, writeall, istat)
       end if
       if (writeall .or. (kmx > 0)) then
          call prop_set(prop_ptr, 'geometry', 'Kmx', kmx, 'Maximum number of vertical layers')
-         call prop_set(prop_ptr, 'geometry', 'Layertype', Layertype, 'Vertical layer type (1: all sigma, 2: all z, 3: use VertplizFile)')
+         call prop_set(prop_ptr, 'geometry', 'Layertype', Layertype, 'Vertical layer type (1: sigma-layers, 2: z- or z-sigma-layers)')
```

Comment:
To only "expose" research options when a user is actively using them, change the .dia print as follows:
```suggestion
         if (layertype == LAYTP_SIGMA .or. layertype == LAYTP_Z) then
            call prop_set(prop_ptr, 'geometry', 'Layertype', Layertype, 'Vertical layer type (1: sigma layers, 2: z- or z-sigma layers)')
         else if (layertype == LAYTP_POLYGON_MIXED .or. layertype == LAYTP_DENS_SIGMA) then
            call prop_set(prop_ptr, 'geometry', 'Layertype', Layertype, 'Vertical layer type (1: sigma layers, 2: z- or z-sigma layers, 3: use VertplizFile for polygon-based mixed layering, 4: density-controlled sigma layers)')
```